docs/run_heudiconv.md

```diff
@@ -17,7 +17,7 @@ $ xnat-heudiconv [OPTIONS] PROJECT SUBJECT BIDS_ROOT_DIR
 **Options**:
 
 * `-S, --session-suffix TEXT`: Suffix of the session for BIDS defaults to 01.              This will produce a session label of sess-01.              You likely only need to change the default for multi-session studies  [default: 01]
-* `--log-id TEXT`: ID or suffix to append to logfile, If empty, date is appended  [default: 01-14-2021-22-10-32]
+* `--log-id TEXT`: ID or suffix to append to logfile, If empty, date is appended  [default: 01-21-2021-15-04-30]
```

Comment:
This date bumping is kinda annoying. It adds noise to each PR. Is there a way to hardcode the date and not have it update?

Reply:
I know, I've been thinking about that too. The default value for the log name is `datetime.now().strftime("%m-%d-%Y-%H-%M-%S")` which I think it's useful to have as a default to guarantee logs don't overwrite themselves. It has been useful for debugging peoples code

I guess, what I could do is leave it empty on the `typer` argument and then in the code check if empty give it the date. Code-wise that is less clean, but maybe less annoying? What do you think?

Reply:
> I guess, what I could do is leave it empty on the `typer` argument and then in the code check if empty give it the date. Code-wise that is less clean, but maybe less annoying? What do you think?

I think that's fine. I don't have any problem with defaults being set in the actual code, but if you think it's much cleaner this way I can deal with a little noise. I just thought I'd check in about it.

Reply:
The other option is to make [this](https://github.com/brown-bnc/xnat-tools/blob/6b7da85b70c2b50c717469efe4b6dba06e5cd5a8/.github/workflows/dev-docs.yml#L30) a bit smarter

A minimal change could be to only run it if the main CLI files have changed. So in this PR it would only touch the post-process doc. 

another idea would be to do some kind of regex on the file and replace the dates with a string (current time formatted as "%m-%d-%Y-%H-%M-%S")... maybe that's the most work but the best?

Reply:
@broarr Alright, I went for the sed option. That way, the `-help` of the CLI still outputs the correct default, but the docs don't add noise

Reply:
Sorry I added more noise, but needed to make small changes to the files to test the triggering of the workflow